### PR TITLE
[onecc-docker]Add tests for onecc-docker

### PR DESCRIPTION
**Pull request by holawan**
This PR adds tests onecc-docker
The test directory helps to test `onecc-docker` command line tool by installing prepare_test_materials.sh script and generating runtestall.sh script automatically.

ONE-DCO-1.0.-Signed-off-by: Dongwan Kim <asdf134652@gmail.com>

draft : #9783 
issue: https://github.com/Samsung/ONE/issues/9712

/cc [@Samsung/ootpg](https://github.com/orgs/Samsung/teams/ootpg)

### CODE REVIEW

**Comment by seanshpark** [comment]
@jyoungyun can you please add a comment of how long this test takes ?

**Comment by seanshpark** [comment]
I had a short offline discussion with @jyoungyun and have some concerns of this change
- our CI is running in Jenkins inside Docker. That is building and running Docker inside a Docker. Technically I think this is possible but not sure currently it is within our CI
- how long would it take of running test? current codes don't run the test. I didn't check the draft yet.
- I think we need some policy about the changes

Thus I don't want to land this PR as of now but after some on-line discussion with an issue.

File: compiler/onecc-docker/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+add_subdirectory(tests)
```
Comment (seanshpark) [inline compiler/onecc-docker/CMakeLists.txt:1]: please add check if test is enabled.

File: compiler/onecc-docker/tests/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        DESTINATION test/onecc-docker)
+
+install(FILES ${PREPARE_TEST_MATERIALS_SH}
+        PERMISSIONS OWNER_WRITE OWNER_READ OWNER_EXECUTE
+                    GROUP_READ GROUP_EXECUTE
+                    WORLD_READ WORLD_EXECUTE
+        DESTINATION test/onecc-docker)
+
+install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/README.txt
+        DESTINATION test/onecc-docker)
```
Comment (seanshpark) [inline compiler/onecc-docker/tests/CMakeLists.txt:68]: test is not executed as of now. this is one of the reason not to land as of now.

File: compiler/onecc-docker/tests/onecc-docker_002.test [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+trap_err_onexit()
+{
+  echo "${filename_ext} FAILED"
+  exit 255
+}
+
+trap trap_err_onexit ERR
+
+workflowfile="onecc-docker_002.workflow.json"
```
Comment (seanshpark) [inline compiler/onecc-docker/tests/onecc-docker_002.test:30]: `workflow` is WIP. let's remove this test.

File: compiler/onecc-docker/tests/prepare_test_materials.sh [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+# limitations under the License.
+
+# See https://github.com/Samsung/ONE/issues/4155 for information
+
+SCRIPT_PATH="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+pushd $SCRIPT_PATH > /dev/null
+
+if [[ ! -s "inception_v3.pb" ]]; then
+    rm -rf inception_v3_2018_04_27.tgz
+    wget https://storage.googleapis.com/download.tensorflow.org/models/tflite/model_zoo/upload_20180427/inception_v3_2018_04_27.tgz
```
Comment (seanshpark) [inline compiler/onecc-docker/tests/prepare_test_materials.sh:24]: `inception_v3_2018_04_27.tgz` is a bit huge. As I know model itself is not important of this feature. let's choose a small one.

File: compiler/onecc-docker/tests/onecc-docker_002.test (hunk 3 above)
Comment (jyoungyun) [reply compiler/onecc-docker/tests/onecc-docker_002.test:30]: The `workflow` is not ready to use yet. So we don't need to implement it.

**Pull request closed without merging**